sandbox/cgroup/freezer.go

```diff
@@ -150,6 +150,11 @@ func applyToSnap(snapName string, action func(groupName string) error, skipError
 		if !strings.HasPrefix(info.Name(), canary) {
 			return nil
 		}
+		// snap applications end up inside a cgroup related to a
+		// service, or when ran standalone, a scope
+		if ext := filepath.Ext(info.Name()); ext != ".scope" && ext != ".service" {
```

Comment:
for clarity, some of the files that we end up skipping here are like:

What about snaps that are in quota groups? It feels like they should be inside the .service one, since I see cgroups of the form `/sys/fs/cgroup/systemd/snap.foo.slice/snap.go-example-webserver.webserver.service/`, so we would eventually end up in the .service when we go deeper, but just wondering if there can be a case where the process is under the `<foo>.slice` directly and not under the nested `<svc>.service`

Reply:
If that were the case then there's no way we can track the process. Perhaps with v1, we have the freezer controller where the place the processes onto a cgroup that we crafted, but with v2 it's all in one tree, so if process is moved out of the group we loose track of it.

Reply:
sorry I only realized now that my comment was malformed, I started to type one question and deleted only half of it when I meant to fully delete it, whoops... In any case you ended up answering the question that we could not track the process if it is not under the nested .service unit so :shrug: